cassandra/connection.py

```diff
@@ -799,9 +801,9 @@ def set_keyspace_blocking(self, keyspace):
                              consistency_level=ConsistencyLevel.ONE)
         try:
             result = self.wait_for_response(query)
-        except InvalidRequestException as ire:
+        except InvalidRequest as ire:
```

Comment:
This was a bug from before, here we received a `InvalidRequest` not an `InvalidRequestException`.